com.unity.postprocessing/PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -1182,6 +1182,9 @@ int RenderBuiltins(PostProcessRenderContext context, bool isFinalPass, int relea
             context.uberSheet = uberSheet;
             context.autoExposureTexture = RuntimeUtilities.whiteTexture;
             context.bloomBufferNameID = -1;
+            int xrActiveEyeBackup = context.xrActiveEye;
+            if (context.stereoRenderingMode == PostProcessRenderContext.StereoRenderingMode.MultiPass)
+                context.xrActiveEye = eye;
```

Comment:
Since depth-of-field and TAA also use xrActiveEye, can you also test/double-check those effects to make sure they still work in multipass with this change?

Reply:
I changed the code so that it's only applied to autoexposure. 
It shouldn't affect t'as and depth of field